chore: prepare 1.7.0 (#629)
redhat-developer/intellij-openshift-connector: README.md

## New features

### Install Helm Charts on the current cluster within the IDE

We're Happy to announce that we've implemented a new action that provide the capability to install
a [Helm](https://helm.sh/) chart into the current cluster.

> Below a small demo using Heml stable repo (https://charts.helm.sh/stable) and installing kuberos chart.

Reviewer: ```suggestion
Below is a small demo of the workflow: installing `kuberos` chart from the Helm stable repo (https://charts.helm.sh/stable) 
```

![](images/1.7.0/helm.gif)

> There is no need to install anything prior to use Helm. By default if there is no repository configured, the openshift

Reviewer: ```suggestion
There is no need to install anything prior to use Helm. By default if there is no repository configured, the OpenShift
```

> helm repository will be use (https://charts.openshift.io/).

Reviewer: ```suggestion
Helm repository will be used (https://charts.openshift.io/).
```
